src/components/displaysettings/displaysettings.template.html

```diff
@@ -63,7 +63,7 @@ <h2 class="sectionTitle">
 
     <div class="selectContainer fldDateTimeLocale hide">
         <select is="emby-select" class="selectDateTimeLocale" label="${LabelDateTimeLocale}">
-            <option value="">${AutoBasedOnLanguageSetting}</option>
+            <option value="">${Auto}</option>
```

Comment:
I feel like it's clearer to let it be "Auto (based on language setting)", though I'd remove the "Auto" part.

Reply:
I was mainly trying to reduce the amount of strings I think, because `auto` already implies automatic detection using some metric and we don't explain any similar instances.